queries.sql

```diff
@@ -6,3 +6,32 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- SQL script that implement transaction 
```

Comment:
- [ ] I can't seem to find any of the below queries in your submission. Kindly follow the project requirement step by step and add each step of the query in your queries.sql file

![image](https://github.com/ezraermy/vet_clinic/assets/63473584/ca6c399b-aed5-4207-bdee-32b0cba300b1)

Reply:
Hi @vickymarz Thank you very much for your helpful and insightful feedback which surely makes my project better.